Code/Tools/FBuild/FBuildTest/Tests/TestDistributed.cpp

```diff
@@ -373,6 +375,35 @@ void TestDistributed::WarningsAreCorrectlyReported_Clang() const
     TEST_ASSERT( GetRecordedOutput().Find( "warning: unused variable 'x' [-Wunused-variable]" ) );
 }
 
+// WarningsAreCorrectlyReported_Clang
+//------------------------------------------------------------------------------
```

Comment:
This seems to only test disitributed compilation, but doesn't test local compilation.